src/main/java/org/apache/maven/xinclude/stax/XIncludeStreamReader.java

```diff
@@ -70,9 +71,19 @@
 @SuppressWarnings("checkstyle:MissingSwitchDefault")
 class XIncludeStreamReader extends StreamReaderDelegate {
 
-    private static final String XINCLUDE_NAMESPACE = "http://www.w3.org/2001/XInclude";
-    private static final String XINCLUDE_INCLUDE = "include";
-    private static final String XINCLUDE_FALLBACK = "fallback";
+    public static final String XINCLUDE_NAMESPACE = "http://www.w3.org/2001/XInclude";
+    public static final String XINCLUDE_INCLUDE_ELEMENT = "include";
+    public static final String XINCLUDE_FALLBACK_ELEMENT = "fallback";
+    public static final String XINCLUDE_LOCAL_ATTRIBUTES = "http://www.w3.org/2001/XInclude/local-attributes";
+    public static final String XINCLUDE_HREF_ATTRIBUTE = "href";
+    public static final String XINCLUDE_PARSE_ATTRIBUTE = "parse";
+    public static final String XINCLUDE_XPOINTER_ATTRIBUTE = "xpointer";
+    public static final String XINCLUDE_FRAGID_ATTRIBUTE = "fragid";
+    public static final String XINCLUDE_SET_XML_ID_ATTRIBUTE = "set-xml-id";
```

Comment:
I'm not sure where this one is coming from. I've never seen it before. is there a spec that mentions this? Google didn't turn one up.

Reply:
They are defined by the XINclude spec: see https://www.w3.org/TR/xinclude-11/#include_element

Reply:
Ah, got added in version 1.1. Thanks. I think I'd stop paying much attention to XInclude by then.